server/router.go

```diff
@@ -13,13 +13,13 @@ func defineRoutes(router *gin.Engine) {
 	v1 := router.Group("/api/v1")
 
 	v1.POST("/signup", userRegistration)
-	user := router.Group("/dashboard")
+	user := v1.Group("/dashboard/")
 	user.Use(authMiddleware.MiddlewareFunc())
-	user.GET("/", getUserFirstName)
+	user.GET("/:id", getUserFirstName)
```

Comment:
After the accessing the dashboard. It doesn't make sense to have other endpoints like `/regparking` and `/regSpot` grouped within the dashboard.

Reply:
Everything is inside the dashboard and after discussing with Felix, I changed to this format!